### PR TITLE
5 training setup

### PR DESCRIPTION
**Pull request by pzal**

### CODE REVIEW

**Review by mateuszwyszynski — APPROVED** [review]
I added some comments before you pushed some new comments, so they are likely outdated. I was able to run fine-tuning and recompute normalisation stats, so it seems that everything works fine. I believe we can merge now, but let's verify that the fine-tuning gives correct results when we have good enough checkpoints.

File: README.md [hunk 1]
```diff
@@ -177,7 +177,7 @@ We will collect common issues and their solutions here. If you encounter an issu
 | Training runs out of GPU memory           | Make sure you set `XLA_PYTHON_CLIENT_MEM_FRACTION=0.9` before running training to allow JAX to use more GPU memory. You can also try reducing the batch size in your training config.        |
 | Policy server connection errors           | Check that the server is running and listening on the expected port. Verify network connectivity and firewall settings between client and server.                                            |
 | Missing norm stats error when training    | Run `scripts/compute_norm_stats.py` with your config name before starting training.                                                                                                          |
-| Dataset download fails                    | Check your internet connection. If using `local_files_only=True`, verify the dataset exists locally. For HuggingFace datasets, ensure you're logged in (`huggingface-cli login`).            |
```
[the remark this answers is not in the document]
Comment (pzal) [reply README.md]: It no longer exists in the newer version of LeRobotDataset.

File: src/openpi/training/config.py [hunk 2]
```diff
@@ -544,7 +573,6 @@ def __post_init__(self) -> None:
         data=LeRobotLiberoDataConfig(
             repo_id="physical-intelligence/libero",
             base_config=DataConfig(
-                local_files_only=False,  # Set to True for local-only datasets.
```
[the remark this answers is not in the document]
Comment (pzal) [reply src/openpi/training/config.py]: It no longer exists in the newer version of LeRobotDataset.

File: src/openpi/training/config.py [hunk 3]
```diff
@@ -531,7 +561,6 @@ def __post_init__(self) -> None:
...
```
[the remark this answers is not in the document]
Comment (pzal) [reply src/openpi/training/config.py]: It no longer exists in the newer version of LeRobotDataset.

File: src/openpi/training/config.py [hunk 4]
```diff
@@ -585,9 +613,7 @@ def __post_init__(self) -> None:
                     )
                 ]
             ),
-            base_config=DataConfig(
-                local_files_only=False,  # Set to True for local-only datasets.
-            ),
+            base_config=DataConfig(),
```
[the remark this answers is not in the document]
Comment (pzal) [reply src/openpi/training/config.py]: It no longer exists in the newer version of LeRobotDataset.

File: src/openpi/training/config.py [hunk 5]
```diff
@@ -501,7 +533,6 @@ def __post_init__(self) -> None:
         data=LeRobotLiberoDataConfig(
             repo_id="physical-intelligence/libero",
             base_config=DataConfig(
-                local_files_only=False,  # Set to True for local-only datasets.
```
[the remark this answers is not in the document]
Comment (pzal) [reply src/openpi/training/config.py]: It no longer exists in the newer version of LeRobotDataset.

**Comment by pzal** [comment]
It was still a draft (primarily because the sbatch scripts will most likely change and right now they contain reference to my unique `qos`), but since it might be useful for others, and since @mateuszwyszynski  already took a look at it, I'm gonna merge it.
